### PR TITLE
Hb hw 4

### PR DESCRIPTION
**Pull request by ShemsheyMaxim**

### CODE REVIEW

File: src/main/java/com/cinema/model/MovieSession.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @OneToOne
+    @JoinColumn(name = "cinema_hall_id")
+    private CinemaHall cinemaHall;
+    private LocalDateTime showTime;
```
Comment (mfedenko09) [inline src/main/java/com/cinema/model/MovieSession.java:26]:
```suggestion
    private LocalDateTime showTime;
```
add please here @ JoinColumn(name "show_time")

**Review by mfedenko09 — APPROVED** [review]

File: src/main/java/com/cinema/model/MovieSession.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+@Entity
+@Table(name = "movie_session")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private Movie movie;
+    @OneToOne
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/model/MovieSession.java]: why OneToOne?

File: src/main/java/com/cinema/model/MovieSession.java [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    @JoinColumn(name = "cinema_hall_id")
+    private CinemaHall cinemaHall;
+    @ JoinColumn(name = "show_time")
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/model/MovieSession.java]:
```suggestion
    @Column(name = "show_time")
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/cinema/model/MovieSession.java (hunk 2 above)
Comment (ShemsheyMaxim) [reply src/main/java/com/cinema/model/MovieSession.java]: done

File: src/main/java/com/cinema/model/MovieSession.java (hunk 3 above)
Comment (ShemsheyMaxim) [reply src/main/java/com/cinema/model/MovieSession.java]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**